docs/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "",
+	"icons": [
+		{
+			"src": "\/android-chrome-192x192.png",
+			"sizes": "192x192",
+			"type": "image\/png"
+		},
+		{
+			"src": "\/android-chrome-512x512.png",
+			"sizes": "512x512",
+			"type": "image\/png"
+		}
+	],
+	"theme_color": "#ffffff",
+	"display": "standalone"
+}
```

Comment:
When I first saw this, I thought: "Why are the `/`s escaped?" I didn't even know that `\/` is a valid JSON escape sequence before! According to this [StackOverflow answeer](http://stackoverflow.com/a/1580682):

> Allowing \/ helps when embedding JSON in a <script> tag, which doesn't allow </ inside strings

So the backslashes could be removed. It doesn't really matter, though, I just wanted to share this interesting (or not XD) piece of trivia.

(Also, this file seems to be indented with tabs, while most (all?) other files in this repo are indented with spaces. I guess it doesn't matter, either. It even saves a few bytes down the wire...))